lib/sensu/client/http_socket.rb

```diff
@@ -0,0 +1,208 @@
+require "base64"
+require "em-http-server"
+require "sensu/json"
+require "sensu/utilities"
+require "sensu/api/utilities/transport_info"
+require "sensu/client/utils"
+
+module Sensu
+  module Client
+    # EventMachine connection handler for the Sensu HTTP client"s socket.
```

Comment:
ha! good catch. Note that was copied from the existing socket.rb comments (I copied the whole block comment and adjusted for the HTTP endpoint, so socket.rb is still using the wrong quotes). I'll fix it here, but won't fix it in socket.rb to not muddle the waters. You have commit so I guess it's easy enough for you to fix it there if you want (or let me know if you want socket.rb fixed in this PR even though is unrelated).